mscore/preferences.cpp

```diff
@@ -194,6 +196,9 @@ void Preferences::init(bool storeInMemoryOnly)
             {PREF_UI_PIANOROLL_LIGHT_BG_TEXT_COLOR,                new ColorPreference(QColor("#111111"))},
       });
 
+      QLocale* local = new QLocale();
+      set(PREF_APP_PAGE_UNITS_VALUE, local->measurementSystem() == QLocale::MetricSystem 
+                                   ? int(QPageSize::Millimeter) :  int(QPageSize::Inch));
```

Comment:
This needs to come after
```
_initialized = true;
```

Reply:
Interesting.  I had not seen any issues while debugging and testing.  I just moved it down, but I'm curious what problem this would cause as is.

Reply:
Otherwise it fails the assertion in Preferences::settings().